docker-compose.override.yml

```diff
@@ -0,0 +1,6 @@
+version: "2.4"
+services:
+  garrysmod:
+    ports:
+      - 27015:27015
+      - 27015:27015/udp
```

Comment:
The configuration in the `docker-compose.override.yml` file is applied over and in addition to the values in the `docker-compose.yml` file.

It is useful to split port mappings out for environments like CI, where we don't want concurrent builds to conflict.